packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -131,6 +131,24 @@ export const LabelLeft: StoryFn<typeof FormField> = (props) => {
         </FormLabel>
         <Input defaultValue="Primary Input value" />
       </FormField>
+      <FormField labelPlacement="left" {...props}>
```

Comment:
If we are adding these here, it ruins the symmetry between Label and LabelLeft. We should probably add a story called LabelLeftWithControls

Reply:
updated